lib/components/narrative/simple-realtime-annotation.js

```diff
@@ -1,9 +1,12 @@
-import React, { Component } from 'react'
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
 
-export default class SimpleRealtimeAnnotation extends Component {
-  render () {
-    return <div className='simple-realtime-annotation'>
-      <i className='fa fa-clock-o' /> This trip uses real-time traffic and delay information
-    </div>
-  }
-}
+const SimpleRealtimeAnnotation = () => (
+  <div className='simple-realtime-annotation'>
+    <i className='fa fa-clock-o' />
+    {' '}
```

Comment:
I think adding whitespace is a no-no when using internationalization. If this is for styling purposes to add padding or something, the styling should be changed instead.